core/src/main/java/com/linecorp/armeria/common/util/Sampler.java

```diff
@@ -82,6 +82,38 @@
         return cast;
     }
 
+    /**
+     * Returns a {@link Sampler} that is configured as specified in the given {@code specification} string.
```

Comment:
@narayaruna this is a change that might be interesting to you. Armeria is using specification strings to do sampling not just of tracing, but also whether or not to keep full stack traces.